core/management/commands/populate.py

```diff
@@ -92,7 +92,12 @@ def handle(self, *args, **options):
             raise Exception("Never call this command in prod. Never.")
 
         Sith.objects.create(weekmail_destinations="etudiants@git.an personnel@git.an")
-        Site.objects.create(domain=settings.SITH_URL, name=settings.SITH_NAME)
+
+        site = get_current_site(None)
```

Comment:
```suggestion
        site = Site.objects.get_current()
```
Ca permet de retirer le `None`, ce qui retire un peu de confusion, parce qu'on ne sait pas à quoi il sert sans chercher dans la doc.